src/components/Errors/FormUnavailable.jsx

```diff
@@ -0,0 +1,24 @@
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import ErrorMessage from 'components/Errors/ErrorMessage';
+
+const FormUnavailable = ({wrapper: Wrapper}) => {
+  const intl = useIntl();
```

Comment:
can you do something about the missing coverage?

Reply:
This is already covered right? https://app.codecov.io/gh/open-formulieren/open-forms-sdk/pull/783?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=checks&utm_campaign=pr+comments&utm_term=open-formulieren

Reply:
Ah yeah looks like it, maybe the annotations in github were not updated then!